src/services/iiif-parser.js

```diff
@@ -85,7 +85,7 @@ export function getTileSources(manifest) {
         const resource = canvas.items[0].items[0].body;
         if (resource.type !== "image" && resource.service) {
           tileSources.push({
-            id: resource["id"],
+            id: resource.service[0]["id"],
```

Comment:
This fixes a small bug I noticed where the </> HTML Embed functionality on Aux images wasn't mapping correctly to the IIIF service. `https://github.com/nulib/digital-collections/pull/953`